src/components/DataMessenger/DataMessenger.js

```diff
@@ -833,9 +833,9 @@ export default class DataMessenger extends React.Component {
                   <span style={{ marginLeft: 5 }}>
                     {lang.openConditionLocking}
                   </span>
-                </div>
-              </div>
-              <hr style={{ padding: 0, marginTop: 0, marginBottom: 0 }} />
+                </div> 
+              </div>*/}
+              {/* <hr style={{ padding: 0, marginTop: 0, marginBottom: 0 }} /> */}
```

Comment:
remove

Reply:
can't remove this for same reason as above